### PR TITLE
Updating Nikto & Burp Suite & Scout Suite mappers

### PR DESCRIPTION
**Pull request by charleshu-8**
Signed-off-by: charleshu-mitre <chu@mitre.org>

### CODE REVIEW

File: libs/hdf-converters/src/burpsuite-mapper.ts [hunk 1]
```diff
@@ -125,24 +120,37 @@ export class BurpSuiteMapper extends BaseConverter {
...
-            source_location: {},
-            code: '',
+            impact: {
+              path: 'severity',
+              transformer: impactMapping(IMPACT_MAPPING)
+            },
+            code: {
+              transformer: (vulnerability: Record<string, unknown>): string => {
+                return JSON.stringify(vulnerability, null, 2);
+              }
```
Comment (camdenmoors) [inline libs/hdf-converters/src/burpsuite-mapper.ts:130]:
```suggestion
              transformer: (vulnerability: Record<string, unknown>): string =>  JSON.stringify(vulnerability, null, 2);
```

**Review by camdenmoors — APPROVED** [review]

File: libs/hdf-converters/src/scoutsuite-mapper.ts [hunk 2]
```diff
@@ -220,51 +219,73 @@ export class ScoutsuiteMapper extends BaseConverter {
...
+      transformer: (data: Record<string, unknown>): Record<string, unknown> => {
+        const auxData = _.omit(data, [
+          'account_id',
+          'environment',
+          'partition',
+          'provider_code',
+          'provider_name',
+          'services'
+        ]);
+        if (auxData instanceof Object) {
```
Comment (Amndeep7) [inline libs/hdf-converters/src/scoutsuite-mapper.ts:277]: what is the purpose of this conditional?

**Review by Amndeep7 — CHANGES_REQUESTED** [review]

File: libs/hdf-converters/src/scoutsuite-mapper.ts (hunk 2 above)
Comment (charleshu-8) [reply libs/hdf-converters/src/scoutsuite-mapper.ts:277]: Type guard so the `auxData` variable's data type will be narrowed from `unknown` to `Object` so I can access attributes.

Comment (charleshu-8) [reply libs/hdf-converters/src/scoutsuite-mapper.ts]: It did when I wrote it, but I checked again and it doesn't seem to be an issue now. Removed the type guard and the mapper still works just fine.

**Review by Amndeep7 — APPROVED** [review]

**Pull request merged**